### DISCUSSION TITLE
EV Car battery representation

### DISCUSSION THREAD

**Discussion by ionutgogu** [slipx06/sunsynk-power-flow-card#485] [Ideas]
Hello,
will be nice to be possible to add an battery attached to load (ex. EV Car![Screenshot_20240622_154431_Home Assistant.jpg](https://github.com/slipx06/sunsynk-power-flow-card/assets/11980766/732c8f85-bd55-419b-8731-d4fd15c832f7)

 battery)

**Comment by Roving-Ronin** [comment]
@ionutgogu 

Have you read the documentation ? Either add it as an essential load (where you have indicated it) or add it as a non_essential load, as per diagram below. Define the icon to use the car or charger symbol.

![image](https://github.com/slipx06/sunsynk-power-flow-card/assets/108674933/a62b9d65-a600-4e55-9356-6cded09c4ee8)
